### PR TITLE
libfreehand 0.1.1

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by mistydemeo** [comment]
Test is failing:

```
==> /usr/bin/clang++ test.cpp -o test -I/usr/local/Cellar/librevenge/0.0.2/include/librevenge-0.0 -I/usr/local/Cellar/libfreehand/0.1.1/include/libfreehand-0.1.1 -L/usr/local/Cellar/librevenge/0.0.2/lib -L/usr/local/Cellar/libfreehand/0.1.1/lib -lrevenge-0.0 -libfreehand-0.1.1
test.cpp:1:10: fatal error: 'libfreehand/FreeHandDocument.h' file not found
#include <libfreehand/FreeHandDocument.h>
         ^
1 error generated.
```

File: Library/Formula/libfreehand.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  desc "Interpret and import Aldus/Macromedia/Adobe FreeHand documents"
+  homepage "https://wiki.documentfoundation.org/DLP/Libraries/libfreehand"
+  url "http://dev-www.libreoffice.org/src/libfreehand/libfreehand-0.1.1.tar.xz"
+  sha256 "ec6676d0c63f7feac7801a1fe18dd7abe9044b39c3882fc99b9afef39bdf1d30"
+
+  depends_on "pkg-config" => :build
+  depends_on "boost" => :build
+  depends_on "librevenge"
+  depends_on "icu4c"
+  depends_on "lcms2"
```
Comment (mistydemeo) [inline Library/Formula/libfreehand.rb]: This is an alias; can you use the real name, `little-cms2`?

File: Library/Formula/libfreehand.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  test do
+    (testpath/"test.cpp").write <<-EOS.undent
+      #include <libfreehand/FreeHandDocument.h>
+      int main() {
+        return libfreehand::FreeHandDocument::RESULT_OK;
+      }
+    EOS
+    system ENV.cxx, "test.cpp", "-o", "test",
+                    "-I#{Formula["librevenge"].include}/librevenge-0.0",
+                    "-I#{include}/libfreehand-0.1.1",
```
Comment (mistydemeo) [inline Library/Formula/libfreehand.rb]: Looks like this should be `libfreehand-0.1`?

**Comment by mistydemeo** [comment]
I didn't get the test to work locally after changing the block:

```
test.cpp:3:41: error: no member named 'RESULT_OK' in 'libfreehand::FreeHandDocument'
  return libfreehand::FreeHandDocument::RESULT_OK;
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
```

**Comment by luzpaz** [comment]
@dtardon can you take a look at this and help with fixing the test block ?

File: Library/Formula/libfreehand.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                          "--disable-dependency-tracking",
+                          "--enable-static=no",
+                          "--disable-werror",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cpp").write <<-EOS.undent
+      #include <libfreehand-0.1/inc/FreeHandDocument.h>
```
Comment (mistydemeo) [inline Library/Formula/libfreehand.rb]: I think this should be `libfreehand/FreeHandDocument.h`?

**Comment by mistydemeo** [comment]
Thanks very much for the submission! Pulled! :sparkles:

**Pull request closed without merging**